MauiMacAppWithBroker.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
We already have a Maui dev app that enables the broker on Mac, do we need another one? Maybe delete the old one and keep the new one?

Reply:
Hi @bgavrilMS , can you point me to the "old" MAUI app with broker on Mac?
I saw in this repo "MauiB2C", "MauiAppBasic", "MauiAppWithBroker" contains "net6.0-maccatalyst", 
but in https://github.com/Azure-Samples/ms-identity-dotnetcore-maui, "maccatalyst" is removed.